### DISCUSSION TITLE
ShapeDownload von Punkten Redllining

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#128] [General]
Punkte die in Redlining/Zeichen  erfasst werden, können nicht als Shape heruntergeladen werden.
Es kommt folgende Fehlermeldung:
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/d44d21de-f96e-4826-9908-fa55a7002c07)

**Comment by gis-eni** [comment]
Stimmt, Punkte (die sind ja neu dazu gekommen, vorher gab es nur Symbole) wurden nicht ins Shape übernommen.
Kommt in Build 5.23.5001
